x/evm/simulation/genesis.go

```diff
@@ -4,16 +4,17 @@ import (
 	"encoding/json"
 	"fmt"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
instead of using the params with the sdk.DefaultBondDenom, we should replace it through an `init` function